src/polymer/polymer-element.ts

```diff
@@ -183,7 +183,15 @@ export class ScannedPolymerElement extends ScannedElement implements
     addMethod(this, method);
   }
 
+  applyDescriptionAsJsdoc() {
+    this.jsdoc = jsdoc.parseJsdoc(this.description);
+    this.description = this.jsdoc.description;
+  }
+
   resolve(document: Document): PolymerElement {
+    if (!this.jsdoc) {
+      this.applyDescriptionAsJsdoc();
+    }
     this.applyJsdocDemoTags(document.url);
```

Comment:
I think all of this should go into the body of `resolveElement`, as calling `resolve` shouldn't mutate the `ScannedPolymerElement`. We may call `resolve` many times on the same scanned element, and we don't want it to change every time.